app/models/registration_agency.rb

```diff
@@ -0,0 +1,12 @@
+class RegistrationAgency < ApplicationRecord
+  validates :agency_type, presence: true
+  validates :state, uniqueness: {scope: :agency_type}
+
+  belongs_to :state
```

Comment:
Do we want to use `shoulda_matcher` to test that any validation and association is in place? We haven't discuss about the value that provides, but we use it for other clients

Reply:
I always feel like `shoulda_matcher` is a little overkill and I don't use it, but I can be persuaded if others find value in it.